src.ts/generated/rpc_errors_types.ts

```diff
@@ -0,0 +1,156 @@
+export class TxExecutionError {
+}
+
+export class InvalidTx extends TxExecutionError {
+}
+
+export class NotEnoughBalance extends InvalidTx {
+    public balance;
+    public cost;
+    public signer_id;
+}
+
+export class SignerDoesNotExist extends InvalidTx {
+    public signer_id;
+}
+
+export class Action extends TxExecutionError {
+    public index;
+}
+
+export class ActorNoPermission extends Action {
+    public actor_id;
+    public account_id;
+}
+
+export class InvalidAccessKey extends InvalidTx {
+}
+
+export class NotEnoughAllowance extends InvalidAccessKey {
+    public account_id;
+    public cost;
+    public allowance;
+    public public_key;
+}
+
+export class MethodNameMismatch extends InvalidAccessKey {
+    public method_name;
+}
+
+export class TriesToStake extends Action {
+    public stake;
+    public account_id;
+    public locked;
+    public balance;
+}
+
+export class DeleteAccountStaking extends Action {
+    public account_id;
+}
+
+export class DeleteKeyDoesNotExist extends Action {
+    public public_key;
+    public account_id;
+}
+
+export class InvalidChain extends InvalidTx {
+}
+
+export class InvalidNonce extends InvalidTx {
+    public tx_nonce;
+    public ak_nonce;
+}
+
+export class InvalidSignature extends InvalidTx {
+}
+
+export class InvalidReceiver extends InvalidTx {
+    public receiver_id;
+}
+
+export class ReceiverMismatch extends InvalidAccessKey {
+    public ak_receiver;
+    public tx_receiver;
+}
+
+export class TriesToUnstake extends Action {
+    public account_id;
+}
+
+export class ActionError extends InvalidAccessKey {
+}
+
+export class Expired extends InvalidTx {
+}
+
+export class AddKeyAlreadyExists extends Action {
+    public account_id;
+    public public_key;
+}
+
+export class RentUnpaid extends Action {
+    public amount;
+    public account_id;
+}
+
+export class AccountDoesNotExist extends Action {
+    public account_id;
+}
+
+export class InvalidSigner extends InvalidTx {
+    public signer_id;
+}
+
+export class AccountAlreadyExists extends Action {
+    public account_id;
+}
+
+export class CostOverflow extends InvalidTx {
+}
+
+export class AccessKeyNotFound extends InvalidAccessKey {
+    public public_key;
+    public account_id;
+}
+
+export class CreateAccountNotAllowed extends Action {
+    public account_id;
+    public predecessor_id;
+}
+
+export class DeleteAccountHasRent extends Action {
+    public balance;
+    public account_id;
+}
+
+export const CLASSMAP = {
```

Comment:
Not sure whether this is needed. Why not just use module itself as a map?

```
import * as CLASSMAP from 'rpc_errors_types'
```

Reply:
Addressed! Thanks!